subprojects/core/src/integTest/groovy/org/gradle/api/tasks/ReproducibleArchivesIntegrationTest.groovy

```diff
@@ -52,14 +54,14 @@ class ReproducibleArchivesIntegrationTest extends AbstractIntegrationSpec {
 
         where:
         input << [
-            ['dir1/file11.txt', 'dir2/file22.txt', 'dir3/file33.txt'].permutations(),
+            ['DIR1/FILE11.txt', 'dir2/file22.txt', 'DIR3/file33.txt', 'dir4/FILE44.txt'].permutations(),
```

Comment:
Adding `dir4` increases the number of tests from 12 to 48. I don't think doing this adds any more reassurance. Let's keep this list to three items.